src/main/connection.cpp

```diff
@@ -12,24 +12,26 @@
 #include "duckdb/execution/operator/persistent/buffered_csv_reader.hpp"
 #include "duckdb/parser/parser.hpp"
 
+
+
 namespace duckdb {
 using namespace std;
 
-Connection::Connection(DuckDB &database) : db(database), context(make_unique<ClientContext>(database)) {
+ Connection::Connection(DuckDB &database) : db(database), context(make_unique<ClientContext>(database)) {
```

Comment:
Could you run the formatter (`python3 scripts/format.py master)`? It seems that the indentation here has been changed.